tests/unit/test_client.py

```diff
@@ -67,16 +67,24 @@ def test_w_emulator(self):
         client_class = mock.Mock()
         emulator_host = emulator_channel = object()
         credentials = _make_credentials()
+
+        client_options = mock.Mock()
+        transport = mock.Mock()
+
         client = _Client(
             credentials, emulator_host=emulator_host, emulator_channel=emulator_channel
         )
         client_info = client._client_info = mock.Mock()
-
-        result = self._invoke_client_factory(client_class)(client)
+        result = self._invoke_client_factory(
+            client_class, client_options=client_options, transport=transport
+        )(client)
 
         self.assertIs(result, client_class.return_value)
         client_class.assert_called_once_with(
```

Comment:
note. I think this test is nearly meaningless now as the call to create the client is of the same signature now. the difference happens within a deeper method than before. I altered this test to function, though it is very similar to the wo_emulator test now.